src/gfx/graphics/webGpu/core/buffer/GPUBufferBase.ts

```diff
@@ -18,6 +18,9 @@ import { FloatArray } from "../../../../../components/matrix/WasmMatrix";
  * @group GFX
  */
 export class GPUBufferBase {
+    static BuffersNeedingReset: Set<GPUBufferBase> = new Set<GPUBufferBase>();
+    static ResetEveryNEncoderUsageCount = 10000;
```

Comment:
Nit: usually UpperCamelCase is for class names in JS/TS; use lowerCamelCase

```suggestion
    static buffersNeedingReset: Set<GPUBufferBase> = new Set<GPUBufferBase>();
    static resetEveryNEncoderUsageCount = 10000;
```

Reply:
alright, changed. I usually go for CamelCase for statics to make it easier to understand they are not class fields - from C#. Not needed with TS as you can't use them without class name even in declaring class - so it's obvious those are statics